src/fast_check/transform.rs

```diff
@@ -903,6 +921,101 @@ impl<'a> FastCheckTransformer<'a> {
     Ok(())
   }
 
+  fn transform_arrow(
+    &mut self,
+    n: &mut ArrowExpr,
+    parent_id_range: Option<SourceRange>,
+  ) -> Result<(), Box<FastCheckDiagnostic>> {
+    let range = parent_id_range.unwrap_or_else(|| n.range());
+
+    let mut clear_body = true;
+
+    if n.return_type.is_none() {
+      if n.is_generator {
```

Comment:
Do we have a test for this?

Reply:
I just realized generator arrow functions are not actually a thing. What is this even lol - why does SWC have this prop in the AST hahaha

Reply:
Oh, I didn't realize this was for an arrow function. Often invalid states are stored in the ast for better diagnostics.